scripts/gceworker.sh

```diff
@@ -14,6 +14,9 @@ source build/shlib.sh
 export CLOUDSDK_CORE_PROJECT=${CLOUDSDK_CORE_PROJECT-${GCEWORKER_PROJECT-cockroach-workers}}
 export CLOUDSDK_COMPUTE_ZONE=${GCEWORKER_ZONE-${CLOUDSDK_COMPUTE_ZONE-us-east1-b}}
 
+DISABLE_COCKROACHDB=${DISABLE_COCKROACHDB-false}
+DISABLE_MANAGED_SERVICE=${DISABLE_MANAGED_SERVICE-false}
```

Comment:
Question. Is this a case of [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)? Is anyone actually going to use this?

What if in this PR just always installs everything and, if people say it now takes too long to spin up a new VM, then we add in this feature (or the appropriate feature given feedback)? This way, if no one uses this, then we won't need to maintain it and if people _do_ ask for something else because it's taking too long to spin up a VM, then we can build the thing they are asking for.

Reply:
Probably yes! 
My intention of having this feature was not to optimise the vm spin time. It was rather done to avoid burdening vm with additional files and tools, which, I agree, is also not something bothering.
I will remove this for now and will wait for someone to actually ask for this.